features/currency/managing_exchange_rates/editing_exchange_rate.feature

```diff
@@ -0,0 +1,25 @@
+@managing_exchange_rates
+Feature: Editing exchange rate
+    In order to modify an exchange rate's configuration
+    As an Administrator
+    I want to be able to edit an exchange rate
+
+    Background:
+        Given the store has currency "US Dollar", "British Pound"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Change exchange rate's ratio
+        Given the store has an exchange rate 1.30 with base currency "US Dollar" and counter currency "British Pound"
+        And I want to edit this exchange rate
+        When I change ratio to 3.21
+        And I save my changes
+        Then I should be notified that it has been successfully edited
+        And it should have a ratio of 3.21
+
+    @ui
+    Scenario: Being unable to change currencies
+        Given the store has an exchange rate 1.30 with base currency "US Dollar" and counter currency "British Pound"
+        When I am editing this exchange rate
```

Comment:
@NeverResponse I would say the other way around 😄  IMO we should say `Given I want to ....` and `When I am editing ...`